Makefile

```diff
@@ -1,7 +1,7 @@
 SHELL := /bin/bash
 DOCKER := $(shell which docker)
 DOCKER_COMPOSE := ${DOCKER} compose
-ARGS_TEST ?=
+ARGS_TESTS ?=
```

Comment:
Unrelated?

Reply:
Yup, it's a nit I ran into as I was testing. I think it's small enough to bundle with this?